modules/apps/object/object-test/src/testFunctional/macros/ObjectRootModel.macro

```diff
@@ -0,0 +1,32 @@
+definition {
+
+	@summary = "Bind an object or multiple objects to the root object path"
+	macro bindToRootObject(childObject,rootObjectPath) {
```

Comment:
I don't know why you're using the word  "path".

Reply:
This macro was made consistent with the interface. You can see this in the modal for binding objects:

![Objects-Objects-Objects-Liferay-DXP](https://github.com/liferay-bpm-qa/liferay-portal/assets/66960853/33baa1c3-d95d-47df-830d-a9e8e151bafd)

cc/ @timpak